src/index.ts

```diff
@@ -1,28 +1,22 @@
-'use strict';
 /**
  * Determine whether two objects are conceptually equal
  * https://github.com/yr/is-equal
  * @copyright Yr
  * @license MIT
  */
+export type DebugFn = (str: string, ...rest: Array<any>) => void;
 
 /**
- * Determine if 'obj1' and 'obj2' are conceptually equal,
- * optionally ignoring properties in 'ignore'
- * @param {Object} obj1
- * @param {Object} obj2
- * @param {Array} [ignore]
- * @param {Debug} [debug]
- * @returns {Boolean}
+ * Determine if 'obj1' and 'obj2' are conceptually equal optionally ignoring properties in 'ignoredObjectProps'
  */
-module.exports = function isEqual (obj1, obj2, ignore, debug) {
-  ignore = ignore || [];
+export function isEqual(obj1: any, obj2: any, ignoredObjectProps?: Array<string>, debug?: DebugFn): boolean {
+  ignoredObjectProps = ignoredObjectProps || [];
```

Comment:
Can we use default function parameter for `ignoredObjectProps` instead of this line? Not sure how this looks with typescript.